server/cache.go

```diff
@@ -677,6 +684,9 @@ func (c *clusterInfo) handleRegionHeartbeat(region *RegionInfo) error {
 		}
 		if region.Leader.GetId() != origin.Leader.GetId() {
 			log.Infof("[region %d] Leader changed from {%v} to {%v}", region.GetId(), origin.GetPeer(origin.Leader.GetId()), region.GetPeer(region.Leader.GetId()))
+			if origin.Leader.GetId() == 0 {
+				c.reactiveRegions++
```

Comment:
If PD restarts, it may load regions from etcd, seem the origin leader may be not 0.